src/index.d.ts

```diff
@@ -0,0 +1,97 @@
+
+type charDataLoader = (
+  char: string,
+  onComplete: (char: string) => void,
+  onError?: (error: any) => void,
+) => void;
+
+interface HanziWriterOptions {
+  charDataLoader?: charDataLoader;
+  onLoadCharDataError?: (error: any) => void;
+  onLoadCharDataSuccess?: (char: string) => void;
+  showOutline:? boolean;
```

Comment:
There is a type on this line. Should be ?: instead of  :?

Reply:
oh dear good catch! that is definitely a typo. I'll try to polish this up this week! would be great to get these typed merged finally.